zhaquirks/tuya/__init__.py

```diff
@@ -986,15 +986,25 @@ def handle_cluster_request(
             )
 
 
+MULTIPLIER = 0x0301
+DIVISOR = 0x0302
+
+
 # Tuya Zigbee Metering Cluster Correction Implementation
 class TuyaZBMeteringCluster(CustomCluster, Metering):
     """Divides the kWh for tuya."""
 
-    MULTIPLIER = 0x0301
-    DIVISOR = 0x0302
     _CONSTANT_ATTRIBUTES = {MULTIPLIER: 1, DIVISOR: 100}
 
 
+# Tuya Zigbee Metering Cluster Correction Implementation
+class TuyaZBMeteringClusterWithUnit(CustomCluster, Metering):
+    """Divides the kWh for tuya."""
```

Comment:
I believe that the UOM is `kW` (and not `kWh`):
https://github.com/home-assistant/core/blob/be899b6ab6bed0ad60655eb611624b0ca237bc10/homeassistant/components/zha/core/channels/smartenergy.py#L218-L223